CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [3.70.0] - 2024-08-21
```

Comment:
As this is not yet released, the section should be named `Unreleased` and there should not be any date. See http://keepachangelog.com/

```suggestion
## [Unreleased]
```